### PR TITLE
Add `-j` flag to `test-amp-challenge-bins.sh` and update ignore list

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
Requires https://github.com/lifting-bits/lifting-tools-ci/pull/13 and https://github.com/lifting-bits/lifting-tools-ci/pull/15

### CODE REVIEW

**Review by 2over12 — APPROVED** [review]
lgtm

**Comment by tetsuo-cpp** [comment]
Sorry @2over12, I need a re-approval to merge since I had to push more commits.

I'll need to fix the squashed commit message before merging since there's so much noise in this PR.

**Review by 2over12 — APPROVED** [review]

**Comment by tetsuo-cpp** [comment]
@2over12 Hmm, same problem again. I had to update the branch since there were new commits to `patchable-ir-main` but can't merge anymore since I've added a merge commit since you approved. Can we consider relaxing that restriction?

**Pull request merged**
